drivers/solis.c

```diff
@@ -479,7 +479,7 @@ static void scan_received_pack(void) {
 	if (!SourceFail && InversorOn)
 		OutFreq = InFreq;
 
-	if (AppPower <= 0) /* charge pf */
+	if (AppPower < 0) /* charge pf */  // •••Sean••• not sure...
```

Comment:
Makes sense with `if( AppPower == 0 )` just a bit below.

Still...  @aquette @zykh @clepple : any memories on this one?

Reply:
never worked on solis